SU-SWS/stanford_basic
# D8CORE-2494 Add "Back to Top" button

# READY FOR REVIEW

# Summary
- Add a "Back To Top" button
- Only displays if the scroll position is over 3 times the current window height.

# Need Review By (Date)
- 5/20

# Urgency
- low

# Steps to Test
1. checkout this branch and clear caches
2. scroll down on a long page
3. click the "Back to top" in the bottom right
4. see that it takes you to the page top.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
